components/textbox.jsx

```diff
@@ -148,6 +147,12 @@ export default class Textbox extends React.Component {
         textbox.blur();
     };
 
+    setInputCursorPosition = (position) => {
+        const textbox = this.refs.message.getTextbox();
+        textbox.selectionStart = position;
+        textbox.selectionEnd = position;
+    }
```

Comment:
We do have `setCaretPosition` in `utils/utils.jsx`. Not sure if it would be better to use that